pkg/config/config.go

```diff
@@ -157,6 +157,9 @@ type PluginConfig struct {
 	//
 	// NOTE: JTERRY75 - This is a hack! DO not submit as the final solution.
 	AutoManageVHDTemplatePath string `toml:"auto_manage_vhd_template_path" json:"autoManageVHDTemplatePath"`
+	// NvidiaGpuVhdPath is the path on disk to the nvidia gpu vhd to mount into an lcow uvm when
+	// a container requests a nvidia gpu
+	NvidiaGpuVhdPath string `toml:"nvidia_gpu_vhd_path" json:"nvidiaGpuVhdPath"`
```

Comment:
Can we put this as part of the config.toml instead via the Opts struct in hcsshim. Basically make this a shim opt. Then we can checkin the default path as part of the toml but not part of the config that gets used for everything?

Reply:
Done